src/openms/include/OpenMS/FORMAT/HANDLERS/MzMLHandler.h

```diff
@@ -5143,6 +5190,10 @@ namespace OpenMS
         if (j == 0)
         {
           os << "\t\t\t\t\t\t<cvParam cvRef=\"MS\" accession=\"MS:1000016\" name=\"scan start time\" value=\"" << spec.getRT() << "\" unitAccession=\"UO:0000010\" unitName=\"second\" unitCvRef=\"UO\" />\n";
+          if (spec.getDriftTime() > 0.0)
```

Comment:
in 3926 you write  if (precursor.getDriftTime() >= 0.0)
here its if (spec.getDriftTime() > 0.0)
intended?